ibu-imager/common/consts.go

```diff
@@ -18,13 +18,35 @@ package common
 
 // Common constants mainly used by packages in ibu-imager
 const (
-	VarFolder = "/var"
-	BackupDir = "/var/tmp/backup"
+	VarFolder       = "/var"
+	BackupDir       = "/var/tmp/backup"
+	BackupCertsDir  = "/var/tmp/backupCertsDir"
+	BackupChecksDir = "/var/tmp/checks"
+
 	// ImageRegistryAuthFile is the pull secret. Written by the machine-config-operator
-	ImageRegistryAuthFile             = "/var/lib/kubelet/config.json"
-	KubeconfigFile                    = "/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/lb-ext.kubeconfig"
-	DefaultRecertImage                = "quay.io/edge-infrastructure/recert:latest"
-	EtcdStaticPodFile                 = "/etc/kubernetes/manifests/etcd-pod.yaml"
-	EtcdStaticPodContainer            = "etcd"
+	ImageRegistryAuthFile = "/var/lib/kubelet/config.json"
+	KubeconfigFile        = "/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/lb-ext.kubeconfig"
+
+	DefaultRecertImage     = "quay.io/edge-infrastructure/recert:latest"
+	EtcdStaticPodFile      = "/etc/kubernetes/manifests/etcd-pod.yaml"
+	EtcdStaticPodContainer = "etcd"
+
 	InstallationConfigurationFilesDir = "/usr/local/installation_configuration_files"
 )
+
+// FilesToRemove is the list of systemd service units, scripts, and env file
+// to be cleaned up after creating the seed image
+var FilesToRemove = []string{
+	"/etc/systemd/system/installation-configuration.env",
+	"/var/usrlocal/bin/installation-configuration.sh",
```

Comment:
Can we read files from the folder and remove accordingly ? I would like not to use list of files if it is possible

Reply:
Sort of same as https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/cmd/create.go#L133 
just need to take file name and build path accordingly

Reply:
agree, will do it as suggested in #79 :)